packages/amplify-category-notifications/index.js

```diff
@@ -25,7 +21,6 @@ module.exports = {
   console,
   deletePinpointAppForEnv,
   initEnv,
-  initEnvPush,
```

Comment:
Why is this removed? I beleive this would break a few things including migration?

Reply:
that was an unneeded leftover from last year, we were talking about possibly separating initEnv locally and then push, but that's not the workflow we used.